core/common/src/main/java/alluxio/exception/ExceptionMessage.java

```diff
@@ -59,32 +61,41 @@
   FAILED_SKIP("Failed to skip {0}"),
   INSTREAM_CANNOT_SKIP("The underlying BlockInStream could not skip {0}"),
   READ_CLOSED_STREAM("Cannot read from a closed stream"),
-  SEEK_NEGATIVE("Seek position is negative: {0}"),
-  SEEK_PAST_EOF("Seek position is past EOF: {0}, fileSize: {1}"),
+  SEEK_NEGATIVE("Seek position is negative: {0,number,#}"),
+  SEEK_PAST_EOF("Seek position is past EOF: {0,number,#}, fileSize: {1,number,#}"),
 
   // netty
-  BLOCK_WRITE_ERROR("Error writing blockId: {0}, sessionId: {1}, address: {2}, message: {3}"),
+  BLOCK_WRITE_ERROR(
+      "Error writing blockId: {0,number,#}, sessionId: {1,number,#}, address: {2}, message: {3}"),
   NO_RPC_HANDLER("No handler implementation for rpc message type: {0}"),
   UNEXPECTED_RPC_RESPONSE("Unexpected response message type: {0} (expected: {1})"),
-  WRITER_ALREADY_OPEN("This writer is already open for address: {0}, blockId: {1}, sessionId: {2}"),
-  UNDER_FILE_WRITE_ERROR("Error writing to under file system fileId: {0}, addr: {1}, msg: {2}"),
+  WRITER_ALREADY_OPEN(
+      "This writer is already open for address: {0}, blockId: {1,number,#}, sessionId: {2,number,"
```

Comment:
could we split in a different way so that the `{2,number,#}` isn't broken between lines?